scripts/training/train_embonly.sh

```diff
@@ -39,7 +39,7 @@ LAUNCHER="accelerate launch \
     --main_process_ip "$MASTER_ADDR" \
     --main_process_port $MASTER_PORT \
     --num_processes $WORLD_SIZE \
-    --machine_rank \$SLURM_PROCID \
```

Comment:
Sorry for the late reply here but why did you remove the `\`? 🤔 Did it not work with the `\`? I think it may have been on purpose to avoid immediate evaluation of the env variable but rather have it be evaluated later

Reply:
The code worked without the `\`. Is it better with the `\`?
I couldn't test the code as it is since I'm using modified Slurm environment thus `sbatch` was working a bit weirdly.

Reply:
Did the code fail with the `\`?